janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -259,7 +259,7 @@
     public static final ConfigOption<Boolean> ADJUST_LIMIT = new ConfigOption<>(QUERY_NS,"smart-limit",
             "Whether the query optimizer should try to guess a smart limit for the query to ensure responsiveness in " +
                     "light of possibly large result sets. Those will be loaded incrementally if this option is enabled.",
-            ConfigOption.Type.MASKABLE, true);
+            ConfigOption.Type.MASKABLE, false);
```

Comment:
After updating configuration options we need to run `mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am` to regenerate configuration references documentation.
Probably we need to make it more clear

Reply:
My bad, forgot about that. Thanks for reminder.